pql/matching.py

```diff
@@ -375,6 +375,13 @@ def handle_Constant(self, node):
         return node.value
     def handle_Num(self, node):
         return node.n
+    def handle_UnaryOp(self, node):
+        op_type = type(node.op)
+        if (op_type == ast.USub):
+            return - node.operand.value
```

Comment:
```suggestion
        op_map = {ast.USub: '-', ast.UAdd: '+'}
        op_class = node.op.__class__
        if op_class in op_map:
            return ast.literal_eval(f'{op_map.get(op_class, "")}{node.operand.value}')
```
파이선 3.8 이상에서는 node.operand 가 `ast.Num` 대신 항상 `ast.Constant` 가 됩니다.  따라서 node.operand.value 도 `1` 대신 `'1'` 이 되는 경우를 발견했습니다. 우선 3.8 을 지원하기 위해 제안을 드려봐요. 테스트러너를 추가해보겠습니다. (_ _)

Reply:
하위 호환성에 문제가 있을 것 같아 조금 더 살펴보겠습니다

Reply:
https://docs.python.org/ko/3/library/ast.html#ast.literal_eval

파이썬 문서에서 literal_eval을 확인했어요 아까 문서를 보다가 덜 본 부분이 있었어요 

ast.UAdd인 경우면 +가 안들어가야해서 지금처럼 해두었는데 
ast.literal_eval에 담갔다가 나오면 +가 사라지네요 

제안해주신 코드로 바꾸는게 더 안전하지 않을까요?